templates/deb/postinst_upgrade.sh.erb

```diff
@@ -31,10 +39,18 @@ after_install() {
 <%=  script(:after_install) %>
 <% end -%>
 
-<% if attributes[:deb_systemd] -%>
+<%# if any systemd services specified, loop through and start them -%>
+<% unless attributes[:deb_systemd].empty? -%>
 systemctl --system daemon-reload >/dev/null || true
-systemctl enable <%= attributes[:deb_systemd] %> >/dev/null || true
-systemctl start <%= attributes[:deb_systemd] %> >/dev/null || true
+sysctl=$(command -v deb-systemd-invoke || echo systemctl)
+<% attributes[:deb_systemd].each do |service| -%>
+<% if attributes[:deb_systemd_enable?]-%>
+systemctl enable <%= service %> >/dev/null || true
```

Comment:
did you mean `$sysctl` instead of `systemctl` here?

Reply:
nope, `deb-systemd-invoke` doesn't have an `enable` option, so `systemctl` is fine to be used http://manpages.ubuntu.com/manpages/bionic/man1/deb-systemd-invoke.1p.html